apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/CommonUtils.cs

```diff
@@ -37,5 +40,21 @@ internal static class CommonUtils
                 .Assembly
                 .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
                 .InformationalVersion;
+
+        internal static Struct CreateServiceContext(string serviceName, string version)
```

Comment:
Let's make this public. Sure, most other code won't use it - but it's not *dangerous* for it to be public, and it avoids the versioning issue.